.github/workflows/test_policy.yml

```diff
@@ -0,0 +1,68 @@
+name: Test Policy
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test_policy:
+    strategy:
+      matrix:
+        dbversion: [ "postgres:latest" ]
+        go: [ "1.17" ]
+        platform: [ ubuntu-latest ] # can not run in macOS and windowsOS
+    runs-on: ${{ matrix.platform }}
+    services:
+      postgres:
+        image: ${{ matrix.dbversion }}
+        env:
+          POSTGRES_PASSWORD: pass
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Build
+        run: go build -v .
```

Comment:
Why do we build it if we're going to `go run main.go`? If we're testing buildability, `go vet` would suffice

Reply:
you are right. was a copy paste from another action.